packages/react-devtools-shared/src/backend/views/TraceUpdates/canvas.js

```diff
@@ -43,8 +43,28 @@ function drawNative(nodeToData: Map<HostInstance, Data>, agent: Agent) {
 }
 
 function drawWeb(nodeToData: Map<HostInstance, Data>) {
+  // if there are no nodes to draw, detach from top layer
+  if (nodeToData.size === 0) {
+    if (canvas !== null) {
+      if (canvas.matches(':popover-open')) {
+        // $FlowFixMe[prop-missing]: Flow doesn't recognize Popover API
+        // $FlowFixMe[incompatible-use]: Flow doesn't recognize Popover API
+        canvas.hidePopover();
+      }
+    }
+    return;
+  }
+
   if (canvas === null) {
     initialize();
+  } else {
+    try {
```

Comment:
Same as above, you don't need it to wrap it in try / catch, since we bumped minimum versions for the extension

Please also check other calls below

Reply:
Thanks for pointing this out! I've removed all try/catch blocks around Popover API calls.